examples/typescript/components/http-hello-world/package.json

```diff
@@ -6,10 +6,10 @@
   "scripts": {
     "build:tsc": "tsc",
     "build:component": "jco componentize -w wit -o dist/http-hello-world.wasm dist/http-hello-world.js",
-    "build:actor": "wash build --sign-only --config-path wasmcloud.toml",
-    "build": "npm run build:tsc && npm run build:component && npm run build:actor",
-    "actor:start": "wash start actor file://$(realpath dist/http_hello_world_s.wasm) --auction-timeout-ms 10000 --timeout-ms 10000",
-    "actor:stop": "wash stop actor typescript-http-hello-world",
+    "build:wasmcloudcomponent": "wash build --sign-only --config-path wasmcloud.toml",
```

Comment:
Since the `jco` step actually builds JS *from* a component, what do you think about renaming this step `build:js` or something of that sort?

Then we could have `build:component` instead of `build:wasmcloudcomponent` which is a little more accurate and even easier to type out.

It would also be nice to have `component:start`, etc.

Reply:
Hello @vados-cosmonic, thanks for the feedback. I updated the PR with your suggestion. Please let me know if I missed anything.

Reply:
Hey @nitame thanks for the update -- this feels much better!